cg/store/crud/create.py

```diff
@@ -415,3 +416,13 @@ def link_case_to_order(order_id: int, case_id: int):
         session = get_session()
         session.execute(insert_statement)
         session.commit()
+
+    def add_illumina_flow_cell(self, flow_cell: IlluminaFlowCell):
+        if not self.get_illumina_flow_cell_by_internal_id(flow_cell.internal_id):
+            session = get_session()
+            session.add(flow_cell)
+            session.commit()
```

Comment:
If possible, I would like us to commit only at the end of the execution. Ie when everything has been added. Ideally all flow cells, but more likely after the flow cell, flow cell metrics and flow cell sample metrics have been added